src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -26,14 +26,21 @@
           Condition="'$(StringResourcesPath)'!='' AND '$(OmitResources)'!='true'"
           Inputs="$(StringResourcesPath)"
           Outputs="$(IntermediateResOutputFileFullPath)">
-    
-    
 
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
         AssemblyName="$(AssemblyName)" />
 
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true'"
+          SourceFiles="$(StringResourcesPath)"
+          DestinationFiles="$(ResourcesFolderPath)\FxResources.$(AssemblyName).SR.resw"
+          SkipUnchangedFiles="true" />
+
+    <Delete Condition="'$(BuildingUAPVertical)' == 'true' AND Exists('$(CommonPriFile)')"
```

Comment:
I'm not sure it is a good idea to do this for every project given the common file is shared and we build in parallel